### DISCUSSION TITLE
More persistent "unsaved changes" indicator when creating/editing a document

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#4388] [Feature Requests & Ideas]
Currently, it's hard to remember that the form is unsaved as there is no indicator that the form has been modified. It would be great if there could be an indicator prompting the user to save.

![image](https://github.com/payloadcms/payload/assets/440015/e2c52321-9890-4bae-a715-9d3286117ca9)

**Comment by DanRibbens** [comment]
I'm not seeing the screenshot you uploaded for whatever reason. Are you get the warning before leaving without saving changes? That has always been our strategy, that along with the ability to click the save button has been all we need.

Another option for you is to enable. If you have collections that you want to use `versions.drafts.autosaves`, you could enable that and get the benefits of autosaving while editing. Of course you may not want drafts and publishing turned on, but it could be something to consider.

**Reply by tyteen4a03** [reply]
@DanRibbens Have updated the post.
